res/blink.js

```diff
@@ -147,4 +147,9 @@
 
   Blink.click = click;
 
+
+  // Window creation callback
+  if (typeof callback_id !== 'undefined') {
+    Blink.sock.onopen = ()=>{ cb(callback_id, true); }
+  }
```

Comment:
This weirded me out a tad. Do you think this is the best place to mark the window as having finished loading? I _think_ so, since by this point it has finished loading all of its blink-related JS, which seems to be the last thing Electron does after creating the window.

But are there any async-y sorts of things I might be missing? Is there a better place to put this that you can think of?

Reply:
This seems fine for now -- even if anything is running async at this point communication between Julia and Electron is possible so it makes sense to return from the `Window()` call.

Reply:
Okay excellent. Thanks!